Notify/Notify/Notify/Views/ProfilePage.xaml.cs

```diff
@@ -12,9 +12,9 @@ public ProfilePage()
         {
             InitializeComponent();
             BindingContext = new ProfilePageViewModel();
-            ((ProfilePageViewModel)BindingContext).LocationButtonCommand.CanExecuteChanged += (s, e) => profileCarouselView.IsVisible = true;
-            ((ProfilePageViewModel)BindingContext).BlueToothButtonCommand.CanExecuteChanged += (s, e) => profileCarouselView.IsVisible = true;
-            ((ProfilePageViewModel)BindingContext).WifiButtonCommand.CanExecuteChanged += (s, e) => profileCarouselView.IsVisible = true;
+            // ((ProfilePageViewModel)BindingContext).LocationButtonCommand.CanExecuteChanged += (s, e) => profileCarouselView.IsVisible = true;
```

Comment:
Do you still need it?

Reply:
No, but I don't want to delete anything that is related to the old ProfilePage.